samples/eshop-dapr/services/catalog-api.bicep

```diff
@@ -67,12 +54,11 @@ resource catalogApi 'Applications.Core/containers@2023-10-01-preview' = {
         ASPNETCORE_ENVIRONMENT: 'Development'
         ASPNETCORE_URLS: 'http://0.0.0.0:80'
         RetryMigrations: 'true'
-        SeqServerUrl: seqRoute.properties.url
+        SeqServerUrl: 'http://seq:5340'
```

Comment:
Is there a way to remove this hardcoding?

Reply:
Applies to all hardcoded strings

Reply:
The hardcoding comes as part of using DNS-SD and conventionally making the name of service = name of container. @rynowak had suggested to look at it as the DNS hostname of service, instead of a typical hardcoded value.

Reply:
This is desirable hardcoding 👍